README.md

```diff
@@ -14,6 +14,7 @@ Shared service to generate PNG/PDF snapshots of our websites.
 **Parameters:**
 
 - `url` — (**required**) the URL you want to render.
+- `service` - (**recommended**) an alphanumeric identifier for the calling service.
```

Comment:
In all our tickets and other docs, we normally say "requesting service" and I inherited this term from @emmajane. I feel like it's the preferred term.